examples/MMF-activity-recognition/handler.py

```diff
@@ -0,0 +1,140 @@
+from abc import ABC
```

Comment:
Was there an example from the MMF repo that you used that helped you write this? Would be good to have a link at the very top in case MMF makes breaking changes in the future

Reply:
@msaroufim it was from an old version of the handler.